freqtrade/tests/conftest.py

```diff
@@ -37,10 +37,12 @@ def log_has_re(line, logs):
 
 def patch_exchange(mocker, api_mock=None, id='bittrex') -> None:
     mocker.patch('freqtrade.exchange.Exchange._load_markets', MagicMock(return_value={}))
+    mocker.patch('freqtrade.exchange.Exchange.validate_pairs', MagicMock())
     mocker.patch('freqtrade.exchange.Exchange.validate_timeframes', MagicMock())
     mocker.patch('freqtrade.exchange.Exchange.validate_ordertypes', MagicMock())
     mocker.patch('freqtrade.exchange.Exchange.id', PropertyMock(return_value=id))
     mocker.patch('freqtrade.exchange.Exchange.name', PropertyMock(return_value=id.title()))
+    # mocker.patch('freqtrade.exchange.Exchange.markets', PropertyMock(return_value={}))
```

Comment:
I think this should work if you put the `markets` fixture as return value... 
should save us from a ton of "stupid" mocking ...

Reply:
So, I agree with you that's exactly what should be done. However, I think that might mess up some of the other tests, that assume that markets are not loaded.. Not sure about that tho